scan-citrix-netscaler-version.py

```diff
@@ -0,0 +1,324 @@
+2022-11-28 11:55:05+00:00,1669636505,953fae977d4baedf39e83c9d1e134ef1,12.1-55.291
+2022-11-30 11:42:25+00:00,1669808545,f063b04477adc652c6dd502ac0c39a75,12.1-65.25
+2022-12-14 15:54:39+00:00,1671033279,14c6a775edda324764a940cfd3da48cb,13.0-89.7
+2023-01-24 17:44:35+00:00,1674582275,c2b8537eb733844f1e0cc4f63210d016,13.0-90.7
+2023-02-22 13:31:29+00:00,1677072689,b4c220db03ea18bc2eebb40e9ad3f4f8,13.1-42.47
+2023-04-05 06:57:33+00:00,1680677853,0b2a3cb74b5c6adbe28827e8b76a9f64,12.1-55.296
+2023-04-12 08:05:14+00:00,1681286714,6925fba74320b9bfb960299f7c3e7cce,13.1-45.61
+2023-04-17 18:09:24+00:00,1681754964,cdb72bd7677da8af9942897256782c9b,13.1-37.150
+2023-04-19 15:34:38+00:00,1681918478,281b46a105662de06fb259293aa79f2a,13.0-90.11
+2023-04-26 11:42:55+00:00,1682509375,1487b55f253ea54b1d3603cc1212f164,13.1-45.62
+2023-04-28 20:39:00+00:00,1682714340,a6a783263968040a97e44d7cac55eda6,12.1-65.35
+2023-04-30 08:54:31+00:00,1682844871,d72c9f2af7ccded704862da7486cfef2,13.1-45.63
+2023-05-12 04:49:56+00:00,1683866996,,13.0-91.12
+2023-05-12 07:33:58+00:00,1683876838,14195083e08df261613408eb5cf3b212,13.1-45.64
+2023-05-15 10:23:44+00:00,1684146224,4d63b52cc99fe712f9be5e4795c854e9,13.0-90.12
+2023-06-03 07:35:50+00:00,1685777750,,13.1-48.47
+2023-07-07 15:32:56+00:00,1688743976,,13.0-91.13
+2023-07-07 16:15:10+00:00,1688746510,e72b4f05a103118667208783b57eee3b,
+2023-07-07 16:17:07+00:00,1688746627,46d83b1a2981c1cfefe8d3063adf78f4,13.1-37.159
+2023-07-07 16:29:27+00:00,1688747367,28e592a607e8919cc6ca7dec63590e04,12.1-55.297
+2023-07-10 18:36:31+00:00,1689014191,,13.1-49.13
+2023-07-28 00:25:01+00:00,1690503901,,14.1-4.42
+2023-08-30 07:03:54+00:00,1693379034,,13.0-92.18
+2023-09-15 06:40:36+00:00,1694760036,,14.1-8.50
+2023-09-21 05:25:24+00:00,1695273924,,13.0-92.19
+2023-09-21 06:17:01+00:00,1695277021,,13.1-49.15
+2023-09-21 17:12:48+00:00,1695316368,155a75fb7efac3347e7362fd23083aa5,12.1-55.300
+2023-09-27 12:27:52+00:00,1695817672,,13.1-37.164
+2023-10-18 07:27:04+00:00,1697614024,,13.1-50.23
+2023-11-22 18:19:39+00:00,1700677179,,14.1-12.30
+2023-12-08 19:10:40+00:00,1702062640,,13.1-51.14
+2023-12-14 10:12:36+00:00,1702548756,,13.0-92.21
+2023-12-15 07:26:58+00:00,1702625218,,13.1-51.15
+2023-12-15 09:18:34+00:00,1702631914,,14.1-12.35
+2023-12-18 07:59:52+00:00,1702886392,f6beac6ccd073f5f7c1a64c4c7e24c7e,12.1-55.302
+2024-01-05 04:15:53+00:00,1704428153,,13.1-37.176
+2024-02-08 05:34:51+00:00,1707370491,,14.1-17.38
+2024-02-29 17:31:08+00:00,1709227868,,13.1-52.19
+2024-04-18 21:13:30+00:00,1713474810,,14.1-21.57
+2024-05-01 05:48:44+00:00,1714542524,fe1071e2b14a5b5016d3eb57ddcfc86d,12.1-55.304
+"""
+
+
+def load_version_hashes() -> tuple[dict, dict]:
+    """Return a tuple of two dictionaries: vhash_to_version, vstamp_to_version.
+
+    Returns:
+        vhash_to_version: dict[str, str] - mapping of version hash to version string
+        vstamp_to_version: dict[int, str] - mapping of rdx_en timestamp to version string
+    """
```

Comment:
```suggestion
def load_version_hashes() -> tuple[dict[str, str], dict[int, str]]:
    """Return a tuple of two dictionaries: vhash_to_version, vstamp_to_version.

    Returns:
        vhash_to_version: dict[str, str] - mapping of version hash to version string
        vstamp_to_version: dict[int, str] - mapping of rdx_en timestamp to version string
    """
```
The type hints in the docstring is not strictly necessary when you fully type it in the function signature IMO.